boards/phytec/phyboard_lyra_am62x/doc/phyboard_lyra_am62x_a53.rst

```diff
@@ -20,13 +20,18 @@ Zephyr OS is ported to run on the Cortex®-A53 core.
     - 4KB EEPROM
   - Ethernet
 
-More information about the board can be found at the
-`PHYTEC website`_.
+See the `PHYTEC AM62x Product Page`_ for details.
+
+.. figure:: img/phyCORE-AM62x_Lyra_frontside.webp
+   :align: center
+   :alt: phyBOARD-Lyra AM62x
+
+   PHYTEC phyBOARD-Lyra with the phyCORE-AM62x SoM
 
 Supported Features
 ==================
 
-The Zephyr phycore_am62x_a53 board configuration supports the following hardware
+The Zephyr phyboard_lyra_am62x_am6234_a53 board configuration supports the following hardware
```

Comment:
use proper name with slashes

Reply:
Did you even check your own PR? https://github.com/zephyrproject-rtos/zephyr/pull/70913/commits/477835129407fa80dcd50005209b61f295054213#diff-bb3ed069bee91ffe0d661beedb71369e2c7c1947913136ed4134428b8f51e69eR34 this is literally your second commit in the current version of your PR

Reply:
I know what you mean. I just don't see any reason in cleaning up the history for documentation only, while the output is absolutely fine. No one will benefit from that change... the only thing that will happen is that 3 people have to review and approve again, which is IMO a waste of time for these people.